pkg/restic/common.go

```diff
@@ -254,12 +261,7 @@ func NewPodVolumeRestoreListOptions(name string) metav1.ListOptions {
 // should be used when running a restic command for an Azure backend. This list is
 // the current environment, plus the Azure-specific variables restic needs, namely
 // a storage account name and key.
-func AzureCmdEnv(backupLocationLister velerov1listers.BackupStorageLocationLister, namespace, backupLocation string) ([]string, error) {
-	loc, err := backupLocationLister.BackupStorageLocations(namespace).Get(backupLocation)
-	if err != nil {
-		return nil, errors.Wrap(err, "error getting backup storage location")
-	}
-
+func AzureCmdEnv(loc *velerov1api.BackupStorageLocation) ([]string, error) {
```

Comment:
Changes in this file are all API breaking changes.
For purposes of backwards compatibility, we should create new functions and add a deprecation notice to the old ones.